tracing-subscriber/src/registry/mod.rs

```diff
@@ -1,19 +1,16 @@
-//! **EXPERIMENTAL**: Storage for span data shared by multiple [`Layer`]s.
-//!
-//! This module is experimental. Although potential breaking changes will be
-//! avoided when possible, we reserve the right to make breaking changes to this
-//! module until it is no longer experimental.
-//!
-//! Add the `registry_unstable` feature to your `Cargo.toml` to enable
-//! this module:
-//!
-//! ```toml
-//! [dependencies.tracing-subscriber]
-//! features = ["registry_unstable"]
-//! ```
+//! Storage for span data shared by multiple [`Layer`]s.
```

Comment:
it would be nice to have examples of how to use the registry here...could be added in a follow-up.